public/stylesheets/site/2.0/10-types-groups.css

```diff
@@ -189,11 +189,11 @@ span.series .position + .divider:after {
   content: "\25CF\0020";
 }
 
-p.kudos {
-  background: url("/images/imageset.png") no-repeat -111px -580px;
-  padding: 0.5em 0.5em 0.5em 60px;
-  margin: 2em 0;
-  min-height: 40px;
+p.kudos { 
```

Comment:
Could you get rid of the extra blank spaces at the ends of these lines? (I think it's possible ReviewDog doesn't run on CSS files, so it missed them.)